yt/frontends/amrvac/data_structures.py

```diff
@@ -242,6 +242,9 @@ def _set_code_unit_attributes(self):
         # This is the reason why it uses setdefaultattr: it will only fill the gaps lefts by the "override",
         # instead of overriding them again
 
+        # check for inconsistencies between override_units and amrvac normalisations
+        self._check_override_units_amrvac()
```

Comment:
how about renaming is `_check_override_consistency()` ? It's already clear from your docstring that the method is frontend-exclusive :)